source/linux-install.html.erb

```diff
@@ -0,0 +1,130 @@
+sudo yum install automake autoconf readline-devel ncurses-devel openssl-devel libyaml-devel libxslt-devel libffi-devel libtool unixODBC-devel
+          <% end %>
+          </div>
+          Other distributions may use different package names and package managers; a search for each of the names given should return your distro's equivalent.
+          </p>
+
+          <h3>Install asdf</h3>
+          <p>
+            <% code("bash") do %>
+git clone https://github.com/asdf-vm/asdf.git ~/.asdf
+
+echo -e '\n. $HOME/.asdf/asdf.sh' >> ~/.bashrc
+echo -e '\n. $HOME/.asdf/completions/asdf.bash' >> ~/.bashrc
+source ~/.bashrc
+            <% end %>
+          </p>
+
+          <h3>Install all the plugins</h3>
+          <p>
+            If you have any trouble running these, start a new shell.
+            <% code("bash") do %>
+asdf plugin-add erlang https://github.com/asdf-vm/asdf-erlang.git
+asdf plugin-add elixir https://github.com/asdf-vm/asdf-elixir.git
+asdf plugin-add nodejs https://github.com/asdf-vm/asdf-nodejs.git
+bash ~/.asdf/plugins/nodejs/bin/import-release-team-keyring
+asdf plugin-add postgres https://github.com/smashedtoatoms/asdf-postgres.git
+            <% end %>
+          </p>
+
+          <h3>Install the version of tools we want</h3>
+          <p>
+            <% code("bash") do %>
+asdf install erlang 19.3
+asdf install elixir 1.4.5
+asdf install nodejs 8.2.1
+asdf install postgres 9.6.3
+            <% end %>
+          </p>
+
+          <h3>Set these versions as your defaults</h3>
+          <p>
+            <% code("bash") do %>
+asdf global erlang 19.3
+asdf global elixir 1.4.5
+asdf global nodejs 8.2.1
+asdf global postgres 9.6.3
+            <% end %>
+
+            Once you have a project to work on, you can add specific versions to a file called <code>.tool-versions</code> which asdf uses to choose which version to use when you switch into the project directory by replacing <code>global</code> in the above snippet with <code>local</code>.
+          </p>
+
+          <h3>Start postgres</h3>
+          <p>
+            <% code("bash") do %>
+pg_ctl start
+            <% end %>
+          </p>
+
+          <h2>Install Phoenix</h2>
+          <p>
+            Since we have just installed Elixir for the first time, we will need to install the Hex package manager as well. Hex is necessary to get a Phoenix app running (by installing dependencies) and to install any extra dependencies we might need along the way.
+          </p>
+          <% code("bash") do %>
+  mix local.hex
+          <% end %>
+          <p>
+            Now that we have Hex installed we can install Phoenix
+          </p>
+          <% code("bash") do %>
+  mix archive.install https://github.com/phoenixframework/archives/raw/master/phoenix_new.ez
+          <% end %>
+
+          And we're good to go! Let's go write some elixir!
+        </div>
+        <!-- /#page-wrapper -->
+    </div>
+    <!-- /#wrapper -->
+
+
+<!-- jQuery -->
```

Comment:
@catherine-jones I just grabbed all this stuff from the mac guide. If it's already somewhere in the layout or something, feel free to remove it all.